cmd/secret/cmd/root.go

```diff
@@ -11,6 +12,13 @@ import (
 	"syscall"
 	"time"
 
+	"github.com/golang-migrate/migrate/v4"
+	_ "github.com/golang-migrate/migrate/v4/database/postgres"
+	_ "github.com/golang-migrate/migrate/v4/source/file"
+	_ "github.com/golang-migrate/migrate/v4/source/github"
```

Comment:
Why do you need github source? I haven't seen any usage neither in tests or usage examples